thinkhazard/static/js/report.js

```diff
@@ -1,138 +1,92 @@
 (function() {
 
-  // UTFGrid data
-  var grid;
-  var keys;
-  var data;
-
-  // Map element selector
-  var mapSelector = '#map';
-
-  // Array used as a temporary storage for event offsetX and offsetY
-  var offsets = new Array(2);
-
-  // Show the "overview" list when the division-name link is clicked
-  $('#division-name').on('click', function() {
-    if (hazardType) {
-      showOverview();
-    }
+  var map = new ol.Map({
+    target: 'map',
+    interactions: [],
+    controls: [],
+    layers: [
+      new ol.layer.Tile({
+        source: new ol.source.Stamen({
+          layer: 'watercolor'
+        }),
+        opacity: 0.5
+      })
+    ],
+    view: new ol.View({ })
   });
-
-  // Change the mouse cursor when an administrative division is detected
-  var currentCursor;
-  $('#map').on('mousemove', function(e) {
-
-    var w = $(this).width();
-    var h = $(this).height();
-
-    if (!currentCursor) {
-      currentCursor = $(this).css('cursor');
-    }
-
-    getEventOffsets(e, offsets);
-
-    var xRelative = offsets[0] / w;
-    var yRelative = offsets[1] / h;
-
-    var data = getDataForPosition(xRelative, yRelative);
-
-    if (data && currentCursor != 'pointer') {
-      currentCursor = 'pointer';
-      $(this).css('cursor', currentCursor);
-    } else if (!data && currentCursor != 'auto') {
-      currentCursor = 'auto';
-      $(this).css('cursor', currentCursor);
-    }
+  map.getView().fitExtent(app.divisionBounds, map.getSize());
+
+  var extent = map.getView().calculateExtent(map.getSize());
+
+  var styleFn = function(feature) {
+    return [new ol.style.Style({
+      fill: new ol.style.Fill({
+        color: 'rgba(0, 0, 0, .1)'
+      }),
+      stroke: new ol.style.Stroke({
+        color: '#000000',
+        width: 1
+      }),
+      text: new ol.style.Text({
+        text: feature.get('name'),
+        scale: 0.9,
+        stroke: new ol.style.Stroke({
+          color: 'rgba(255, 255, 255, 0.6)',
+          width: 5
+        }),
+        fill: new ol.style.Fill({
+          color: 'black'
+        })
+      })
+    })];
+  };
```

Comment:
For performance reasons we should create the styles array and the static styles outside the style function. But we can do this with a separate PR, when we add the choropleths.